docs/source/getting_started.rst

```diff
@@ -64,6 +64,16 @@ This example runs `ASHA <tutorials/multifidelity/mf_asha.html>`__ with
 seconds on the local machine it is called on
 (:code:`trial_backend=LocalBackend(entry_point=entry_point)`).
 
+Experimentation with Syne Tune
+==============================
+
+If you plan to use advanced features of Syne Tune, such as different execution
+backends or running experiments remotely, writing launcher scripts like
+``examples/launch_height_simple.py`` can become tedious. Syne Tune provides an
+advanced experimentation framework, which you can learn about in
+`this tutorial <tutorials/experimentation/README.html>`__, or also in
+`this one <tutorials/odsc_tutorial/README.html>`__.
```

Comment:
Looking at `examples/launch_height_simple.py`, I see it uses LocalBackend: https://github.com/awslabs/syne-tune/blob/main/examples/launch_height_simple.py#L16. Isn't that in fact a good reference example? Or is there an alternative approach that we recommend instead?

Do any of the examples in the `examples/` folder that show usage of the experiment launchers? 

Maybe a more general question - what do these experimentation launchers provide that the backends themselves don't already provide directly?  https://github.com/awslabs/syne-tune/tree/main/syne_tune/backend

And would anything stop us from adding those features to the backends directly, rather than creating this new folder under `syne_tune.experiments.launchers.` and putting launcher scripts there.

Otherwise, it seems like there are two competing ways of doing things. And from a user experience point of view, it may not be clear what the recommended path is.

Reply:
I don't really understand this.

Reply:
The idea of `syne_tune.experiments.launchers` is for users not to have to write complex launcher scripts for every case. This has nothing to do with backends, which are just a part of the definition of a tuning experiment.

Reply:
There are two competing ways of doing things. One is most flexible, but very hard to run complicated experiments. The other is much simpler and allows to quickly obtain simple launcher scripts with CLI.

Reply:
The launcher scripts in `examples/` are bare bones, and error prone. You see how often things are wrong with them. They are not useful to run many experiments. They are basically using the essentials, which is OK, but tedious for most users.

The stuff in `syne_tune/experiments` is for users who do not care to create a scheduler with a searcher and a backend, and make sure they all fit together with their training function. Each backend has its own ways in which you need to create them, it is quite complex. With `experiments`, this is independent of internals, you just specify things about your training function.

Reply:
I am a bit tired of these discussions. Please have a look at Optuna, where they provide simple ways of describing a study and then execute it. We should always strive for simplicity.